tests/parsers/test_angular.py

```diff
@@ -97,3 +99,33 @@ def test_parser_return_footer_from_commit_message():
 def test_parser_should_accept_message_without_scope():
     assert angular_parser("fix: superfix")[0] == 1
     assert angular_parser("fix: superfix")[3][0] == "superfix"
+
+##############################
+# test custom parser options #
+##############################
+@mock.patch("semantic_release.history.parser_angular.config.get",
+            wrapped_config_get(angular_parser_default_level_bump='50'))
+def test_parser_custom_default_level():
+    assert angular_parser("test(parser): Add a test for angular parser")[0] == 50
+
+
+@mock.patch("semantic_release.history.parser_angular.config.get",
+            wrapped_config_get(
+                angular_parser_allowed_types=
+                'custom,build,chore,ci,docs,fix,perf,style,refactor,test'))
+def test_parser_custom_allowed_types():
+    assert angular_parser("custom: ...")[0] == 0
+    assert angular_parser("custom(parser): ...")[1] == "custom"
+    pytest.raises(UnknownCommitMessageStyleError, angular_parser, "feat(parser): ...")
+
+
+@mock.patch("semantic_release.history.parser_angular.config.get",
+            wrapped_config_get(angular_parser_minor_types='docs'))
+def test_parser_custom_minor_types():
+    assert angular_parser("docs: write some docs")[0] == 2
```

Comment:
Don't you need to also set the allowed types

Reply:
`docs` and `test` are supported by default, so for those tests they should only need what's there. i did try adding it for docs, just to confirm, and the test still failed the same way.

for the custom one, i added `custom` and removed `feat`, but i didn't change the default bump level. if `custom` was allowed it should return 0 and not throw an exception (but it is throwing an exception)

for `feat` since it was removed, it _should_ throw an exception, but if you comment out the assertions about `custom`, the test fails because it _doesn't_ throw an exception